src/SmartComponents/PatchSetWizard/WizardAssets.js

```diff
@@ -84,8 +84,9 @@ export const schema = (patchSetID) => ({
                     fields: [
                         {
                             name: 'systems',
-                            component: 'review-systems',
-                            validate: [{ type: 'validate-systems' }]
+                            component: 'review-systems'
+                            //We can use this later in case UX wantes to prevent patch sets without zero systems
```

Comment:
I think it should be discussed if such a feature will be wanted, before we add it

Reply:
This is already existing code. I did not want to remove it since UX might change their mind. Let's leave this code for now.